.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,30 @@
-#### Read our contributing.md if you haven't already: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md
+# If this is your first PR, or not, take the time to read our CONTRIBUTING.md file! You can see it here: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md
 
-# General Documentation
 
-### Intent of your Pull Request
+# Github documenting your Pull Request
 
-Explain what the purpose of your PR is. Mention if you tested your changes. If you changed a map, make sure you used the mapmerge tool. Mention if your PR is related to an issue.
-You can type "Fixes issue #6" and issue 6 will be automatically closed when your pull request is merged.
+Remove this text and explain what the purpose of your PR is.
 
-### Why is this change good for the game?
+Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
+If this is an Issue Correction, you can type "Fixes Issue #6" to link the PR to the corresponding Issue number #6.
 
-# Wiki Documentation
-
-### Briefly describe your PR and the impacts of it, in layman's terms. 
-This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.
-
-### What should players be aware of when it comes to the changes your PR is implementing?
+Prefix the PR title with [admin] if it involves something admin related. 
+Prefix the PR title with [s] if you are fixing an exploit, so that it is not announced on the Yogstation Discord and the server.
 
-### What general grouping does this PR fall under? 
-For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?
+Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying.
 
-### Are there any aspects of the PR that you would like us not to mention on the Wiki?
+# Wiki Documentation
 
-### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
-For example, "This new chem will heal X brute damage". 
+Remove this text and write all information regarding your changes that should be known and documented through the Yogstation Wiki. 
+Important documentation information includes, but is not limited to: any numberical values that have been changed, any images that have to be updated, names of specific pages that will be impacted by your changes. 
 
 # Changelog
 
-Edit this changelog for changes that are noticeable by the players. Remove it if this isn't the case. If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name. Prefix the PR title with [admin] if it's something admin related. Prefix the PR title with [s] if you are fixing an exploit so it is not announced on discord and the server.
+Edit the changelog at the bottom for changes that are noticeable by the players. Remove it if this isn't the case.
```

Comment:
We require a changelog for all changes just fyi

Reply:
Good point, I just copy pasted that from Rob's changes, so I'll modify it